src/components/Main/Main.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Main.css';
+import MovieItem from './MovieItem/MovieItem';
+
+interface movieBaseType { id: string, name: string, year: string, genre: string, image: string }
+
+const Main = ({ movieBase } : any) => (
```

Comment:
Why does it have `any` type? Btw, turn on linter rule that restricts usage of `any`.